asserts/device_asserts.go

```diff
@@ -44,6 +44,16 @@ func (mod *Model) Model() string {
 	return mod.HeaderString("model")
 }
 
+// ModelDisplayName returns the display-friendly name of the model or
+// falls back to Model if this was not set.
+func (mod *Model) ModelDisplayName() string {
+	display := mod.HeaderString("model-display-name")
```

Comment:
@emgee how strange does model-title sound for a native speaker, it sounds a bit strange to me?
@niemeyer what about @emgee idea of simple using display-name?